sql/builtin_functions.cc

```diff
@@ -2169,6 +2202,41 @@ BoundFunction upper(const std::vector<BoundSqlExpression> & args)
 
 static RegisterBuiltin registerUpper(upper, "upper");
 
+BoundFunction flatten(const std::vector<BoundSqlExpression> & args)
+{
+    // Return an expression but with the timestamp modified to something else
+
+    if (args.size() != 1)
```

Comment:
The catch lower in the call stack will print the function name, by the way, so we dont need to repeat it.